PolyEngine/Engine/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+find_package(OpenGL REQUIRED)
+if(WIN32)
+	find_package(GLEW REQUIRED PATHS ThirdParty/glew/lib/Release/ PATH_SUFFIXES Win32 x64) #FIXME(vuko): does this even work? how to make it search default paths first? (according to docs it should by default, but it fails on my system as opposed to invocation without given paths)
```

Comment:
Oh my, that is one clean CMake script; I could learn from that. Well, what I wanted to do is to search the system path first, then fall back on the included one. And since the CMake docs confuse the hell out of me, this is the result. I'm still not sure about the given example, though, please advise.